src/main.js

```diff
@@ -0,0 +1,23 @@
+function convertItemsToUnorderedList(listItems) {
+  const ulList = document.createElement("ul");
```

Comment:
I suggest renaming this to `ulElement` (or something like that), as `ulList` essentially means "unordered list list", which is too verbose 😂.